src/chi/components/bulk-actions/bulk-actions.scss

```diff
@@ -0,0 +1,117 @@
+@import '_global-variables';
+@import '_global-mixins';
+
+.chi-bulk-actions {
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  padding: 0.5rem;
+
+  @include respond-to(md) {
+    background-color: $bulk-actions-background-color;
+    justify-content: space-between;
+    padding: 0.25rem 0.875rem 0.25rem 1.5rem;
+  }
+
+  .chi-divider {
+    &.-vertical {
+      display: none;
+
+      @include respond-to(md) {
+        border-right-color: $bulk-actions-divider-border-right-color;
+        display: block;
+        height: $bulk-actions-divider-height;
+      }
+    }
+  }
+
+  &__start,
+  &__end {
+    align-items: center;
+    display: flex;
+  }
+
+  &__end {
+    display: none;
+
+    @include respond-to(md) {
+      display: block;
+    }
+  }
+
+  &__top,
+  .chi-bulk-actions__buttons-mobile {
+    background-color: $bulk-actions-background-color;
+    display: flex;
+    left: 0;
+    position: fixed;
+    width: 100%;
+
```

Comment:
Can we add z-index here so the data table cells will not overlap bulk actions in mobile view?